modules/custom/az_media/css/az-media-edit.css

```diff
@@ -9,3 +9,31 @@
   width: 400px;
   max-width: 100%;
 }
+
+figure[data-view-mode="az_natural_size"] .az-media.embed-responsive {
```

Comment:
I think these styles need to be updated from `embed-responsive` to `ratio`.

Reply:
Thanks, i was trying to understand why this wasn't working.